Index: gst-plugins-good1.0/ext/qt6/meson.build
===================================================================
--- gst-plugins-good1.0.orig/ext/qt6/meson.build
+++ gst-plugins-good1.0/ext/qt6/meson.build
@@ -60,7 +60,7 @@ if not qt6qml_dep.found()
   subdir_done()
 endif
 
-qsb = find_program('qsb-qt6', 'qsb', required: qt6_option)
+qsb = find_program('qsb-qt6', 'qsb', dirs: [ '/usr/lib/qt6/bin/' ], required: qt6_option)
 if not qsb.found()
   subdir_done()
 endif
